lib/services.js

```diff
@@ -8,8 +8,8 @@ async function startSyslogService (udid, socket) {
   return new SyslogService(await startService(udid, SYSLOG_SERVICE_NAME, socket));
 }
 
-async function startWebInspectorService (udid, socket) {
-  return new WebInspectorService(await startService(udid, WEB_INSPECTOR_SERVICE_NAME, socket));
+async function startWebInspectorService (udid, version, socket) {
```

Comment:
is this function public? if yes then I would rather add the new argument as the last one or bump the major module version

Reply:
It is. I'm actually thinking about finding the OS version here so the api wouldn't change